### PR TITLE
Announcing Scala.js 1.0.0-RC2.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+**We encourage all users who are able to do so to test their projects with this RC, and report any issue as soon as possible.**
+
+As the change in major version number witnesses, this release is *not* binary compatible with 0.6.x, nor with the previous milestones and RCs of the 1.x series.
+Libraries need to be recompiled and republished using this RC to be compatible.
+Moreover, this release is not entirely source compatible with 0.6.x either.
+
+These release notes contain cumulative changes with respect to 0.6.31.
+Compared to 1.0.0-RC1, this release candidate contains the following changes:
+
+* Fix the handling of versions of Scala.js to correctly work post-v1.0.0
```
Comment (gzm0) [inline _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md]: work correctly?

File: _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+* Remove a few `@deprecated` methods that were forgotten in 1.0.0-RC1
+* Internal IR contract change: use IR static methods as the contract to call main methods and static methods in JDK APIs
+* Fix [3888](https://github.com/scala-js/scala-js/issues/3888): linking error with a nested object named `class`
```
Comment (gzm0) [inline _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md]: #3888?

File: _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md (hunk 1 above)
Comment (sjrd) [reply _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md]: Hum I'm bad with placing adverbs in English, but I thought they were supposed to come before the verb, no?

File: _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md (hunk 2 above)
Comment (sjrd) [reply _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md]: Done.

File: _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md (hunk 1 above)
Comment (gzm0) [reply _posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md]: ["to work correctly"][1]: 31e6 Google results
["to correctly work"][2]: 87e3 Google results

:P

[1]: https://www.google.com/search?sxsrf=ACYBGNTLB7QEccmmSeuhFQQccPNl3lDNZA%3A1576171540357&ei=FHjyXZqxFYe9UNexqrAK&q=%22to+work+correctly%22&oq=%22to+work+correctly%22&gs_l=psy-ab.3..0i7i30l4j0i30.9248.12505..12823...0.2..0.279.1002.2-4......0....1..gws-wiz.......0i71j0i5i30.gUZqAU6U4Zs&ved=0ahUKEwiai5LG0LDmAhWHHhQKHdeYCqYQ4dUDCAs&uact=5
[2]: https://www.google.com/search?sxsrf=ACYBGNTLB7QEccmmSeuhFQQccPNl3lDNZA%3A1576171540357&ei=FHjyXZqxFYe9UNexqrAK&q=%22to+work+correctly%22&oq=%22to+work+correctly%22&gs_l=psy-ab.3..0i7i30l4j0i30.9248.12505..12823...0.2..0.279.1002.2-4......0....1..gws-wiz.......0i71j0i5i30.gUZqAU6U4Zs&ved=0ahUKEwiai5LG0LDmAhWHHhQKHdeYCqYQ4dUDCAs&uact=5

**Pull request merged**